### PR TITLE
Add tyler thome GitHub handle 7175

### LINKED ISSUE #7175: Add github-handle for Tyler Thome in home-unite-us.md
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started Page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to create a single variable `github-handle` to hold the github handle for each member of the leadership team. Eventually `github-handle` will replace the `github` and `picture` variables, reducing redundancy in the project file.

### Action Items
- [x] Open the file `_projects/home-unite-us.md` in the IDE
- [x] Replace:
```
- name: Tyler Thome
```
with
```
- name: Tyler Thome
  github-handle:
```
- [x] Do not use a tab to indent `github-handle`. YAML doesn't allow tabs; it requires spaces.
- [x] Using docker, confirm that the appearance of the project webpage is unchanged at all screen sizes. The project webpage URL can be found below under Resources.

### For Merge Team
- [x] Release the dependency for this issue in #5441. If all the dependencies have been completed, close that issue.

### Resources/Instructions
https://github.com/hackforla/website/wiki/project.md-file-template
https://jekyllrb.com/
For QA to validate change: https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md?plain=1
Project Webpage: https://www.hackforla.org/projects/home-unite-us

- This issue is part of #5441.

### PR DESCRIPTION
**Pull request by ihop-56**
Fixes #7175 

### What changes did you make?
- Added the `github-handle:` field under `name: Tyler Thome` in the `_projects/home-unite-us.md` file. This field was added without a value as per the issue's instructions.

### Why did you make the changes?
- The change was made to prepare the project file for future enhancements, reducing redundancy by standardizing the GitHub handle field.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
No visual changes to the website.

### CODE REVIEW

**Comment by bexux** [comment]
Review ETA: 8 PM pacific 8/24/24
Availability: 5-8 PM Friday, Noon - 8PM Saturday

File: _projects/home-unite-us.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ alt: 'Home Unite Us'
 image-hero: /assets/images/projects/home-unite-us-hero.png
 leadership:
   - name: Tyler Thome
+    github-handle: tylerthome
```
Comment (bexux) [inline _projects/home-unite-us.md]: Almost there - Should be `github-handle:` with no value present yet.

**Review by bexux — CHANGES_REQUESTED** [review]
This is very close to being done! Looks like you had the code correct on your first commit, but then the second commit, `Changes before switching branches` added in value that is not needed for this issue. 

The PR description reads well, branching done correctly, and issue attached correctly. Just need the small change.

**Comment by bexux** [comment]
@ihop-56  I noticed you have a second PR - [7309](https://github.com/hackforla/website/pull/7309) pushed up to address issue 7175. I assume this PR is in response to @codyyjxn's feedback to have the issue number in the branch name?

**Comment by ihop-56** [comment]
Yes, that's right

**Comment by ihop-56** [comment]
Hi @bexux,

I've made the requested changes:
- Removed the value from `github-handle` for Tyler Thome as per your feedback.

Please review and let me know if any further adjustments are needed.

Thank you for your guidance!

**Comment by t-will-gillis** [comment]
HI @ihop-56 please don't forget to re-request a review by selecting the chasing arrows next to the reviewer's handle when you make changes to address a reviewer's comments:
Screenshot 2024-08-25 083845

**Review by codyyjxn — APPROVED** [review]
Nice @ihop-56 this looks great

**Review by 8alpreet — APPROVED** [review]
Hi @ihop-56,

Thanks for making changes in accordance to feedback on this and the previous PR. 

### Opinion:
* The PR looks good, but I would encourage you to do one last thing: squash the commits. I'm not sure if this is practice on this project, but merging three commits will pollute the the commit history unnecessarily.

### Nitpick:
* Not necessary, but you could capitalize "Tyler Thome" in the PR title.

I am approving, but urge you to squash the commits.

**Comment by t-will-gillis** [comment]
Hey @8alpreet and @ihop-56 FYI the commits are squashed when we merge the PRs, thus we do not ask anyone to squash their commits. Devs can if they would like to, but since we 'squash and merge' it is not needed for the PR

**Review by bexux — APPROVED** [review]
Looks great! Thank you for making the changes.

**Comment by 8alpreet** [comment]
> Hey @8alpreet and @ihop-56 FYI the commits are squashed when we merge the PRs, thus we do not ask anyone to squash their commits. Devs can if they would like to, but since we 'squash and merge' it is not needed for the PR

Got it, thanks @t-will-gillis!

**Review by kgold2018 — APPROVED** [review]
Hi  @ihop-56 ! Looks good! Thank you for making the changes.

**Pull request merged**